src/tickit/core/components/system_simulation.py

```diff
@@ -28,6 +32,9 @@ class SystemSimulationComponent(BaseComponent):
     #: corresponding output of an internal component.
     expose: Dict[PortID, ComponentPort]
 
+    # a list of all child tasks
+    _tasks: List[asyncio.Task] = field(default_factory=list)
```

Comment:
I would prefer more descriptive variable naming to a comment (e.g. `_child_tasks`)